views/js/review/plugins/navigation/review-panel/tpl/accordionPanel.tpl

```diff
@@ -26,3 +26,23 @@
     </footer>
 {{/if}}
 </div>
+{{#if replaceIcons}}
+<style>
+    {{#each replaceIcons}}
+        .review-panel.accordion .review-panel-content .review-panel-item.item-{{@key}} .review-panel-label:before {
+            color: {{color}};
+        }
+        .item-answer.show-correct.{{@key}} .icon {
+            color: {{color}};
+        }
+        {{#if code}}
+            .review-panel.accordion .review-panel-content .review-panel-item.item-{{@key}} .review-panel-label:before {
+                content: "{{code}}";
+            }
+            .item-answer .icon-{{@key}}:before {
+                content: "{{code}}";
+            }
+        {{/if}}
+    {{/each}}
```

Comment:
**quibble:** Is it required to set the style dynamically? It will reduce the ability to properly theme the UI. It would better be managed by a custom theme.

Reply:
I agree, but first request was that customer need to update icons by configs, because they didn't know what icons should be. Later they provided icons and colors. That's why configuration and not custom theme.
But I will clean this and move to custom theme.